apps/client/src/app/components/admin-market-data/admin-market-data.component.ts

```diff
@@ -241,4 +245,46 @@ export class AdminMarketDataComponent implements OnDestroy, OnInit {
           });
       });
   }
+
+  private openCreateAssetProfileDialog() {
+    this.dataService
+      .fetchAdminMarketData({})
```

Comment:
Hmm, I would not fetch market data here. I noticed it right away because it took over 6 seconds to open the dialog for me 🙈 (yes, there are too many asset profiles without paging)